flashcard_runner.rb

```diff
@@ -0,0 +1,48 @@
+require './lib/card'
+require './lib/turn'
+require './lib/deck'
+require './lib/round'
+require './lib/card_generator'
+
+def start  
```

Comment:
I'm so glad you were able to write your runner to be dynamic enough to handle any number of cards or categories. Great work!